app/src/main/res/navigation/graph_autofill.xml

```diff
@@ -9,17 +9,12 @@
             xmlns:app="http://schemas.android.com/apk/res-auto"
             xmlns:tools="http://schemas.android.com/tools"
             android:id="@+id/graph_autofill"
-            app:startDestination="@id/fragment_null">
+            app:startDestination="@id/fragment_filter_backdrop">
```

Comment:
This looks like it should cause problems when the app is locked. 

I think I'd prefer to keep using blank and then move on to the `AutofillFilterBackdropFragment`. This makes it congruent with the `graph_main.xml`.